opentera-webrtc-native-client/OpenteraWebrtcNativeClient/src/OpenteraAudioDeviceModule.cpp

```diff
@@ -452,12 +452,29 @@ void OpenteraAudioDeviceModule::run()
             &elapsedTimeMs,
             &ntpTimeMs);
 
+        if (elapsedTimeMs == 0)
```

Comment:
I think elapsedTimeMs is always 0. Why have you added this logic?

Reply:
It is incremented by the webrtc function, and it resets when a connection is dropper (is becomes -1), then when a new connection is established (it starts incrementing again but the first one is 0). This logic is there so that new connections refresh the counter, to start anew.
I'm not sure if there is a guaranteed zero though, il will check.

Reply:
You can keep only the else parts because normally NeedMorePlayData is always called every 10 ms. You can check the Alsa implementation of the AudioDeviceModule.